assets/js/components/user-input/UserInputQuestionWrapper.js

```diff
@@ -87,16 +106,20 @@ export default function UserInputQuestionWrapper( props ) {
 							{ __( 'Next', 'google-site-kit' ) }
 						</Button>
 					) }
-					{ back && (
-						<Button
-							tertiary
-							className="googlesitekit-user-input__buttons--back"
-							onClick={ back }
+					{ complete && (
+						<SpinnerButton
+							className="googlesitekit-user-input__buttons--complete"
+							onClick={ complete }
+							isSaving={ isScreenLoading }
+							disabled={ hasErrorForAnswer( values ) }
 						>
-							{ __( 'Back', 'google-site-kit' ) }
-						</Button>
+							{ breakpoint === BREAKPOINT_SMALL
+								? __( 'Complete', 'google-site-kit' )
+								: __( 'Complete setup', 'google-site-kit' ) }
```

Comment:
It would be better to have a single translation string here that had the words "setup" hidden with a `<VisuallyHidden>` tag using `createInterpolateElement()`. The text is more useful/accessible in the full form, and users of assistive technology on an iPhone, for instance, would benefit from the extra text.

Plus: it doesn't require a re-render on size change.

Reply:
I moved to a css solution, let me know what you think.

Reply:
That's totally good—also: my bad for suggesting `<VisuallyHidden>`, since this should only be hidden at mobile sizes. So they new way looks good 👍🏻